### PR TITLE
Branch dev/cross lang framework

### PR DESCRIPTION
**Pull request by hhr293**
Update Readme

### CODE REVIEW

File: cczoo/cross_lang_framework/README.md [hunk 1]
```diff
@@ -82,21 +89,29 @@ GRAMINEDIR=%gramine_repo_local_path% SGX_SIGNER_KEY=%sgx_signer_key_path% make S
...
 cd ~/confidential-computing-zoo/cczoo/cross_lang_framework/clf_client/app
-GRAMINEDIR=/home/ubuntu/gramine SGX_SIGNER_KEY=/home/ubuntu/gramine/Pal/src/host/Linux-SGX/signer/enclave-key.pem make SGX=1
+
+GRAMINEDIR=/home/ubuntu/gramine SGX_SIGNER_KEY=/home/ubuntu/.config/gramine/enclave-key.pem make SGX=1
 ```
 
 ## Run the framework and sample code
-- Launch the server
+- The value of MRSIGNER should be ensured at first
+- Launch the server in one session
```
Comment (liangintel) [inline cczoo/cross_lang_framework/README.md:98]: session->terminal?

File: cczoo/cross_lang_framework/README.md [hunk 2]
```diff
@@ -82,21 +89,29 @@ GRAMINEDIR=%gramine_repo_local_path% SGX_SIGNER_KEY=%sgx_signer_key_path% make S
...
-./clf_server &
+./clf_server 
 ```
-- Launch the sample app (client)
+- Launch the sample app (client) in another session
 ```bash
 cd ~/confidential-computing-zoo/cczoo/cross_lang_framework/clf_client/app
 gramine-sgx java -Xmx8G clf_test
 ```
+- Then from the server session you can see the value of MRSigner, then replace the original value of MRSigner in the clf_server.conf with the output value on the screen.
```
Comment (liangintel) [inline cczoo/cross_lang_framework/README.md:111]: session->terminal?

File: cczoo/cross_lang_framework/README.md [hunk 3]
```diff
@@ -82,21 +89,29 @@ GRAMINEDIR=%gramine_repo_local_path% SGX_SIGNER_KEY=%sgx_signer_key_path% make S
...
+- Kill the clf_server thread and rerun the command above the launch the server again, then go to another session to launch the sample app(client). the java example should success now.
```
Comment (liangintel) [inline cczoo/cross_lang_framework/README.md:112]: the -> to
session->terminal

Comment (hhr293) [reply cczoo/cross_lang_framework/README.md]: correct it.

**Pull request closed without merging**